fuel-tests/Cargo.toml

```diff
@@ -32,10 +35,12 @@ fuel-txpool = { path = "../fuel-txpool" }
 futures = "0.3"
 insta = "1.8"
 itertools = "0.10"
+portpicker = "0.1.1"
```

Comment:
This doesn't seem necessary now that we removed full integ testing of the CLI

Reply:
will remove